sdk/webpubsub/Azure.Messaging.WebPubSub/tests/WebPubSubTestEnvironment.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+
+namespace Azure.Rest.WebPubSub.Tests
+{
+    public class WebPubSubTestEnvironment : TestEnvironment
+    {
+        public string Endpoint => GetRecordedVariable("WPS_ENDPOINT");
```

Comment:
Nit - consider using WEBPUBSUB since WPS won't make sense in two months if you leave one of those on your dev box.
```suggestion
        public string Endpoint => GetRecordedVariable("WEBPUBSUB_ENDPOINT");
```